### PR TITLE
feature: allow monitoring multiple RDS instances

### LINKED ISSUE #5: alarm names should be unique
terraform doesn't complain about the conflict, but the alarms will constantly have their dimension overwritten with every deploy if the names aren't unique.

### PR DESCRIPTION
**Pull request by joe-niland**
## what
* adds instance prefix to SNS alarms

## why
* Otherwise multiple instances overwrite each other

## references
* original authors: @rfvermut, @rebelthor
* closes #5 
* closes #10

### CODE REVIEW

**Comment by JohnShortland** [comment]
@joe-niland Can you run /test bats again as the results have expired, would be interested to see why this is failing.

**Comment by JohnShortland** [comment]
@joe-niland I think its an issue with the test itself, isn't it? Can't configure provider.

**Comment by joe-niland** [comment]
@JohnShortland yes that's true. Looking into it.

**Comment by bg46z** [comment]
bump

File: alarms.tf [hunk 1]
```diff
@@ -10,8 +10,64 @@ locals {
...
+  context    = module.label.context
+}
+
+module "swap_usage_too_high_label" {
+  source     = "cloudposse/label/null"
+  version    = "0.25.0"
+  attributes = ["swap_usage_too_high"]
+  context    = module.label.context
+}
+
```
Comment (jamengual) [inline alarms.tf:33]: is it possible to use a for_each instead?

Comment (joe-niland) [reply alarms.tf:33]: Yep, good idea @jamengual 
Added, and tests run again.

**Review by nitrocode — APPROVED** [review]

**Pull request merged**